azure-pipelines-official.yml

```diff
@@ -109,6 +110,10 @@ extends:
       tsa:
         enabled: true
         configFile: '$(Build.SourcesDirectory)/eng/TSAConfig.gdntsa'
+      binskim:
+        # BinSkim from 1ES analyzes only uploaded outputs by default,
+        # but our outputs are nupkg/zip files and BinSkim cannot process those.
+        analyzeTargetGlob: '$(Build.SourcesDirectory)/artifacts/**.dll'
```

Comment:
> Also does this mean it's not scanning in say `src/` anymore?

It never did, it only scanned the uploaded outptus.

> Presumably when it added suppor for NuPkg we'd need to change this to be everything?

Right. But I think I can actually remove this setting, because the `outputParentDirectory` setting already ensures we scan the whole `artifacts/` dir, so the default binskim pattern in 1ES should work again (this `analyzeTargetGlob` is a leftover from a config that I had before I set the `outputParentDirectory`, but that made binskim scan all artifacts repeatedly on each output upload).